c10/core/TensorImpl.cpp

```diff
@@ -267,7 +267,7 @@ bool TensorImpl::has_storage() const {
 #endif
 
 void TensorImpl::throw_storage_access_error() const {
-  TORCH_CHECK(false, "Cannot access storage of ", tensorimpl_type_name());
+  TORCH_CHECK_NOT_IMPLEMENTED(false, "Cannot access storage of ", tensorimpl_type_name());
```

Comment:
Well this one is not strictly a not implemented error? Maybe more of a TypeError?

But if that makes the whole Meta testing simpler, I understand :D

Reply:
Yeah, this PR is kind of wrong, but it is "easier"; because I then need to go fix the test suite to stop assuming that there is a storage(). If you like I can remove this from the stack and maintain this in my slush patch until I get far enough that I don't need it anymore.

Reply:
Ok I think it is ok if it makes things easier.